### DISCUSSION TITLE
Category Submissions 2

### DISCUSSION THREAD

**Discussion by DaveBowman2001** [celestiamobile/celestia-addon-validator#3665] [General]
Submit a new category if a submitted addon does not properly belong to any of the existing groups on [mobi](https://celestia.mobi/resources/categories)

## For New Categories
If a New Addon does not fit into any existing category, you can propose a new one by following the format below:
```
**ID:** (optional, use only for updating an existing category)
**Short Name**: (used when context about parent category is given somewhere)
**Full Name**:
**Description**:
**Parent**: (N/A if it is a root category)
**Children**: (N/A if it is a leaf category, you might want to change children's short names to be concise)
**Image**: (must be about 900 * 300 pixels in size in order for the whole image to fit)
**Add-ons**: (If it's a split category, provide a list of existing add-ons to move onto the proposed new category. Please don't forget to include their ID numbers as well)
- (Name 1 = Addon ID)
- (Name 2 = Addon ID)
- etc.
```

## Parent Category (Example)
**Full Name**: Exoplanets
**Description**: Artistic Impressions of real exoplanets arranged by confirmation status and discovery years
**Children**: [0 to 9](https://celestia.mobi/resources/category?category=B9B2FE1F-15BD-894B-EDA6-4BB12035625A), [A, B, and C](https://celestia.mobi/resources/category?category=CA784631-C85E-42AF-840C-82DA22C449FF), [D, E and F](https://celestia.mobi/resources/category?category=DFD5BBCD-3A98-48AF-8E08-B2EC3FAA0AAE), [G](https://celestia.mobi/resources/category?category=340AA62C-1E60-40B2-8E86-9F04AD5153AE), [H, I and J](https://celestia.mobi/resources/category?category=B15857BC-ADB9-4E42-87F6-465DA2D2D881), [K](https://celestia.mobi/resources/category?category=0EEF518F-C9C8-E27D-AA9C-8747FF2094B7), [L](https://celestia.mobi/resources/category?category=7A498EF6-BA5B-4734-A6D6-3041DE91B01A), [M and N](https://celestia.mobi/resources/category?category=C5681CE6-249A-4210-8FCB-490EB953B175), [O](https://celestia.mobi/resources/category?category=00AF3056-5512-42F3-A6CC-3B66CF6203EC), [P](https://celestia.mobi/resources/category?category=35D8CF1B-95AD-439C-BED9-7F64E704F606), [Q, R and S](https://celestia.mobi/resources/category?category=BFD9A6FE-6799-467A-AFE0-7152E5AA2C8A), [T, U and V](https://celestia.mobi/resources/category?category=6455338A-F2B9-5FB3-4CB4-5A00A62C7CEB), [W, X, Y and Z](https://celestia.mobi/resources/category?category=6F6AAA53-C204-4856-963C-E720962FDB3D), [Rogue Planets](https://celestia.mobi/resources/category?category=BAE0F719-A734-400B-B19D-D2626B3D186B), [Unconfirmed](https://celestia.mobi/resources/category?category=903EF64A-7C11-23A1-902A-E2F52001BED7)
**Image**: 
![image](https://user-images.githubusercontent.com/95486841/173229519-72bc9847-e3dc-452e-98fb-257e83082eeb.png)

## Children Category (Example)
**Short Name**: Unconfirmed
**Full Name**: Exoplanets: Unconfirmed
**Description**: These addons depict the star systems whose planets were spotted but is yet to be confirmed by follow-up observations
**Parent**: [Exoplanets](https://celestia.mobi/resources/categories?parent=1B0E1953-C21C-D628-7FA6-33A3ABBD1B40)
**Image**: 
![image](https://user-images.githubusercontent.com/95486841/173229467-26927ad3-afb9-43e6-8ae6-75847ea6b954.png)
**Add-ons**:
- 82 Eridani (Unconfirmed Exoplanets) = [DC7F1559-4B6F-C164-CFC1-4AE9701C322A](https://celestia.mobi/resources/item?item=DC7F1559-4B6F-C164-CFC1-4AE9701C322A)
- KOI-4878 = [C1591BB3-A1C3-64D3-E817-28AFA7C63C04](https://celestia.mobi/resources/item?item=C1591BB3-A1C3-64D3-E817-28AFA7C63C04)
- KOI-5737 = [FB57873D-274E-82C0-B35A-42006894FAB5](https://celestia.mobi/resources/item?item=FB57873D-274E-82C0-B35A-42006894FAB5)
- KOI-5819 = [53F67CD7-0D8F-47E0-B216-5AA844366B12](https://celestia.mobi/resources/item?item=53F67CD7-0D8F-47E0-B216-5AA844366B12)
- KOI-7235 = [B7266580-B32B-2DFF-6785-D350E8E76001](https://celestia.mobi/resources/item?item=B7266580-B32B-2DFF-6785-D350E8E76001)
- KOI-8007 = [D67545F5-9CC6-0EDF-E21B-E58BB21F3CEC](https://celestia.mobi/resources/item?item=D67545F5-9CC6-0EDF-E21B-E58BB21F3CEC)
- Kepler-452 = [86E1E9EC-ADF6-C370-CD40-A0C8A151F4E4](https://celestia.mobi/resources/item?item=86E1E9EC-ADF6-C370-CD40-A0C8A151F4E4)
- MACHO 120.22303.5389 = [8BBFAC5E-BDD9-EC1C-8647-661A8475677B](https://celestia.mobi/resources/item?item=8BBFAC5E-BDD9-EC1C-8647-661A8475677B)
- Pollux = [B26ED397-40F0-4135-7337-065602E70542](https://celestia.mobi/resources/item?item=B26ED397-40F0-4135-7337-065602E70542)
- TOI-709 = [AAC9F87E-CDA3-9B67-FB48-E4C380B49DCB](https://celestia.mobi/resources/item?item=AAC9F87E-CDA3-9B67-FB48-E4C380B49DCB)
- Unconfirmed Planets = [07167CDD-7E3B-17CE-12DA-3718A6FC242B](https://celestia.mobi/resources/item?item=07167CDD-7E3B-17CE-12DA-3718A6FC242B)
- V391 Pegasi = [3B3C7732-6D98-59DC-8FEE-A3AEA0D0FC35](https://celestia.mobi/resources/item?item=3B3C7732-6D98-59DC-8FEE-A3AEA0D0FC35)

**Comment by DaveBowman2001** [comment]
Splitting the WXYZ category into two....the reason was that the XYZ-named exoplanet addons got buried deep by the W-named exoplanet addons

**ID:** [6F6AAA53-C204-4856-963C-E720962FDB3D](https://celestia.mobi/resources/category?category=6F6AAA53-C204-4856-963C-E720962FDB3D)
**Short Name**: Confirmed (W)
**Full Name**: Exoplanets: W
**Description**: These are the confirmed star systems with planets that have names starting with W

**Comment by DaveBowman2001** [comment]
**Short Name**: Confirmed (X, Y and Z)
**Full Name**: Exoplanets: X, Y and Z
**Description**: These are the confirmed star systems with planets that have names starting with X, Y, and Z
**Parent**: Exoplanets
**Image**: 
![Untitled2](https://user-images.githubusercontent.com/95486841/203805840-c6b3db61-9c83-4de2-8d3e-7ff5a1bcbaa5.jpg)
**Add-ons**:
- Xi Aquilae = [2012D31C-F9CF-4D90-A8C0-EB0F5BAB35EA](https://celestia.mobi/resources/item?item=2012D31C-F9CF-4D90-A8C0-EB0F5BAB35EA)
- XO-4 = [9DA383D2-4A68-4360-9BFD-40D178C308E6](https://celestia.mobi/resources/item?item=9DA383D2-4A68-4360-9BFD-40D178C308E6)
- XO-7 = [8C4C64D2-402B-4082-9F16-4922B9D1E1BB](https://celestia.mobi/resources/item?item=8C4C64D2-402B-4082-9F16-4922B9D1E1BB)
- YSES 1 = [C45CAE70-69F2-4B08-B2D5-89E04567D9F7](https://celestia.mobi/resources/item?item=C45CAE70-69F2-4B08-B2D5-89E04567D9F7)
- YSES 2 = [3E759839-E59F-4DDC-AE00-962900ABCDE1](https://celestia.mobi/resources/item?item=3E759839-E59F-4DDC-AE00-962900ABCDE1)
- YZ Ceti = [DAC026DB-ADD1-5D72-52D7-2B00453F8D46](https://celestia.mobi/resources/item?item=DAC026DB-ADD1-5D72-52D7-2B00453F8D46)

```
2012D31C-F9CF-4D90-A8C0-EB0F5BAB35EA
9DA383D2-4A68-4360-9BFD-40D178C308E6
8C4C64D2-402B-4082-9F16-4922B9D1E1BB
C45CAE70-69F2-4B08-B2D5-89E04567D9F7
3E759839-E59F-4DDC-AE00-962900ABCDE1
DAC026DB-ADD1-5D72-52D7-2B00453F8D46
```

**Comment by DaveBowman2001** [comment]
Renaming the Exoplanets: TUV category

**ID**: [6455338A-F2B9-5FB3-4CB4-5A00A62C7CEB](https://celestia.mobi/resources/category?category=6455338A-F2B9-5FB3-4CB4-5A00A62C7CEB)
**Short Name**: Confirmed (T)
**Full Name**: Exoplanets: T
**Description**: These are the confirmed star systems with planets that have names starting with T

**Comment by DaveBowman2001** [comment]
**Short Name:** Confirmed (U and V)
**Full Name:** Exoplanets: U and V
**Description:** These are the confirmed star systems with planets that have names starting with U and V
**Parent:** Exoplanets
**Image:**
![Untitled](https://user-images.githubusercontent.com/95486841/225894502-e09fc81a-82a7-45cd-b41f-1493bba4fe56.jpg)
**Add-ons**:
- UKIRT-2017-BLG-001L = [BEF3065E-F55F-4F63-A855-E0BAB4DB9CAB](https://celestia.mobi/resources/item?item=BEF3065E-F55F-4F63-A855-E0BAB4DB9CAB)
- Upsilon Andromedae = [E172F6E7-E470-D886-C2FD-E10D8A2FCE23](https://celestia.mobi/resources/item?item=E172F6E7-E470-D886-C2FD-E10D8A2FCE23)
- VHS J1256-1257 = [BCD60C81-0E16-442B-ADD2-6A4B30E86EDA](https://celestia.mobi/resources/item?item=BCD60C81-0E16-442B-ADD2-6A4B30E86EDA)

```
- BEF3065E-F55F-4F63-A855-E0BAB4DB9CAB
- E172F6E7-E470-D886-C2FD-E10D8A2FCE23
- BCD60C81-0E16-442B-ADD2-6A4B30E86EDA
```

**Comment by levinli303** [comment]
i'll defer these updates until we have items in those categories.

**Comment by DaveBowman2001** [comment]
**Short Name**: Celestia Worldbuilding Project
**Full Name**: Fictional: Celestia Worldbuilding Project
**Description**: A massive worldbuilding project set in the Triangulum Galaxy made by various add-on developers using Celestia since 2020
**Parent**: Fictional
**Image**: 
![cover_image](https://github.com/celestiamobile/celestia.mobi/assets/95486841/919a5214-f0b0-41d3-bbea-2d5894c558c1)
**Add-ons**: N/A

**Comment by SpaceExplorer2008** [comment]
As soon as I finish sending/updating the planets and moons addons, I will start sending the Celestia Origin minor planets catalog (+4600 objects). Given the number of asteroid models, it will have to be sent in several parts.

However, all MBAs, NEOs, TNOs and centaurs are on the same Origin asteroids.ssc. And I'm not the person who will be willing to separate minor planet by minor planet in their respective classifications. Since it would not be appropriate, for example, to include TNOs together in the "Main Belt" category, I am creating a new subcategory dedicated exclusively to this catalog.

@levinli303

**Short Name:** CO's Minor Planet Catalog
**Full Name:** Asteroids & NEOs: CO's Minor Planet Catalog
**Description:** A large collection of minor planets (MBAs, NEOs, TNOs and centaurs) from Celestia Origin project. When complete, the catalog includes more than 4600 objects, with more than 1000 asteroids having shape models based on light curve inversion
**Parent:** Asteroids & NEOs
**Image:** 
![20240302_150304](https://github.com/celestiamobile/celestia-addon-validator/assets/99847641/0e513c2a-c65a-4747-9f3a-193293d1126d)
**Add-ons:** N/A

**Comment by levinli303** [comment]
moving add-on related issues to this repo instead. Please @ me for categories that are ready to be added/modified.

**Comment by levinli303** [comment]
> I chose this image precisely because it is one of Celestia itself.

we should choose more artistic photos to align with others.

**Comment by levinli303** [comment]
> Centaurs and TNOs

I suppose parent is `Solar System` category

**Comment by DaveBowman2001** [comment]
@levinli303 

**Short/Full Name**: Centaurs and TNOs
**Description**: These are all the rocky and icy bodies found within the outer Solar System
**Parent**: Solar System
**Children**: 
- Centaurs
- Trans-Neptunian Objects (TNOs) = [5DABA8EB-7105-8592-2DA8-505AC6584A55](https://celestia.mobi/resources/category?category=5DABA8EB-7105-8592-2DA8-505AC6584A55)

**Image**: 
![Image](https://github.com/user-attachments/assets/75bf7e0b-3500-4cd4-952a-ac35c3c7b2a8)

**Comment by DaveBowman2001** [comment]
@levinli303 

**Short/Full Name**: Centaurs
**Description**: These are rocky/icy bodies that orbits the Sun between Jupiter and Neptune. They usually have characteristics of both asteroids and comets
**Parent**: Centaurs and TNOs
**Image**: 
![Image](https://github.com/user-attachments/assets/84164797-2513-4fd6-a22e-ba2f0e93f637)
**Add-ons**: 
- 2060 Chiron = [77960B1D-1F2B-7681-508D-1457BE0DE363](https://celestia.mobi/resources/item?item=77960B1D-1F2B-7681-508D-1457BE0DE363)
- 7066 Nessus = [1C4926DB-470D-A244-C573-921E898F1C2B](https://celestia.mobi/resources/item?item=1C4926DB-470D-A244-C573-921E898F1C2B)
- 10199 Chariklo = [96E64AEE-EA92-4E00-6A3B-7D8D09E6B7D7](https://celestia.mobi/resources/item?item=96E64AEE-EA92-4E00-6A3B-7D8D09E6B7D7)
- 54598 Bienor = [996A34F1-34E2-4B7A-A2F3-EBC675C5E08E](https://celestia.mobi/resources/item?item=996A34F1-34E2-4B7A-A2F3-EBC675C5E08E)

```
77960B1D-1F2B-7681-508D-1457BE0DE363
1C4926DB-470D-A244-C573-921E898F1C2B
96E64AEE-EA92-4E00-6A3B-7D8D09E6B7D7
996A34F1-34E2-4B7A-A2F3-EBC675C5E08E
```

**Comment by DaveBowman2001** [comment]
@levinli303 

Relocating the TNOs category 

**ID:** [5DABA8EB-7105-8592-2DA8-505AC6584A55](https://celestia.mobi/resources/category?category=5DABA8EB-7105-8592-2DA8-505AC6584A55)
**Description**: These are all the rocky and icy bodies found beyond Neptune's orbit, from the Kuiper Belt up to the Oort Cloud
**Parent**: Centaurs and TNOs

**Comment by DaveBowman2001** [comment]
@levinli303 
Renaming the Exoplanets: HIJ category 

**ID:** [B15857BC-ADB9-4E42-87F6-465DA2D2D881](https://celestia.mobi/resources/category?category=B15857BC-ADB9-4E42-87F6-465DA2D2D881)
**Short Name**: Confirmed (H)
**Full Name**: Exoplanets: H
**Description**: These are the confirmed star systems with planets that have names starting with H

**Comment by DaveBowman2001** [comment]
@levinli303 
**Short Name:** Confirmed (I and J)
**Full Name:** Exoplanets: I and J
**Description:** These are the confirmed star systems with planets that have names starting with I and J
**Parent:** Exoplanets
**Image:**
![Untitled](https://user-images.githubusercontent.com/95486841/225894311-18077f5a-0130-42b9-8a24-02f4aff4db85.jpg)
**Add-ons**:
- Iota Horologii = [D2C9D07B-A865-4BF6-8A74-4045DEB34ED7](https://celestia.mobi/resources/item?item=D2C9D07B-A865-4BF6-8A74-4045DEB34ED7)

```
D2C9D07B-A865-4BF6-8A74-4045DEB34ED7
```

**Comment by DaveBowman2001** [comment]
@levinli303 
Renaming the Asteroids & NEOs category

**ID**: [C789D7FC-F80E-EBDA-94BA-79FBE31FF9C1](https://celestia.mobi/resources/categories?parent=C789D7FC-F80E-EBDA-94BA-79FBE31FF9C1)
**Short/Full Name**: Asteroids and NEOs

**Comment by DaveBowman2001** [comment]
@levinli303 
**Short Name**: Interstellar Objects
**Full Name**: Solar System: Interstellar Objects
**Description**: These are all the asteroids/comets that have entered the Solar System from interstellar space
**Parent**: Solar System
**Image**:
![Image](https://github.com/user-attachments/assets/0f202f6f-e3d3-42d0-a1f9-9fefa6848e5c)
**Add-ons**: 
- 1I/'Oumuamua = [DDF8778A-6B88-2A83-F5BB-F628720598B2](https://celestia.mobi/resources/item?item=DDF8778A-6B88-2A83-F5BB-F628720598B2)
- 2I/Borisov = [1D334C27-F66D-C20E-0115-AE94733A42B9](https://celestia.mobi/resources/item?item=1D334C27-F66D-C20E-0115-AE94733A42B9)
- 3I/ATLAS = [D8442CC8-0D2C-42B6-BD42-959BD2AB0580](https://celestia.mobi/resources/item?item=D8442CC8-0D2C-42B6-BD42-959BD2AB0580)

```
DDF8778A-6B88-2A83-F5BB-F628720598B2
1D334C27-F66D-C20E-0115-AE94733A42B9
D8442CC8-0D2C-42B6-BD42-959BD2AB0580
```

**Comment by DaveBowman2001** [comment]
@levinli303 

**Short Name**: Navigation Satellites
**Full Name**: Spacecraft: Navigation Satellites
**Description**: These are the spacecraft that provide real-time geopositioning and navigation across the globe via GPS/GNSS networks
**Parent**: Spacecraft
**Image**:
![Image](https://github.com/user-attachments/assets/561bdeda-aaf0-4c57-b508-83fb5a587dd2)
**Add-ons**:
- Galileo Navigation Satellite System = [2BC6678C-4B1A-2CFC-15C4-F33D995FE61B](https://celestia.mobi/resources/item?item=2BC6678C-4B1A-2CFC-15C4-F33D995FE61B)
- GLONASS-M = [85A27529-2825-47FC-9D17-EF6CC360B2D3](https://celestia.mobi/resources/item?item=85A27529-2825-47FC-9D17-EF6CC360B2D3)

```
2BC6678C-4B1A-2CFC-15C4-F33D995FE61B
85A27529-2825-47FC-9D17-EF6CC360B2D3
```

**Comment by DaveBowman2001** [comment]
@levinli303 
Splitting the `Comets` category

**Short/Full Name**: Comets
**Description**: Contains additional comets and enhancements to the ones that are included in the stock installation of Celestia
**Parent**: [Solar System](https://celestia.mobi/resources/categories?parent=B2E44BE0-9DF7-FAB9-92D4-F8D323D31250)
**Children**:
- Periodic Comets
- Non-periodic Comets

**Image**: 
![Image](https://github.com/user-attachments/assets/07351257-7505-456d-b2e6-da78be6a6f2d)

**Comment by DaveBowman2001** [comment]
@levinli303 

**ID:** [4F4D674A-6BBE-EC82-3F1A-9CBA4E91044D](https://celestia.mobi/resources/category?category=4F4D674A-6BBE-EC82-3F1A-9CBA4E91044D)
**Short Name**: Periodic Comets
**Full Name**: Comets: Periodic
**Description**: These are comets that have orbital periods less than 200 years in length. This include Encke-type, Jupiter-family, Halley-type, and Chiron-type comets
**Parent**: Comets
**Image**:
![Image](https://github.com/user-attachments/assets/5490bfd6-7ab9-4b20-be67-37c56b767b9f)
**Add-ons**:
- 1P/Halley = [3DCF1DB4-7CDC-436C-80E6-056D30D0DEF1](https://celestia.mobi/resources/item?item=3DCF1DB4-7CDC-436C-80E6-056D30D0DEF1)
- 2P/Encke = [4A311B53-D159-48C4-A950-E8E63A2B6996](https://celestia.mobi/resources/item?item=4A311B53-D159-48C4-A950-E8E63A2B6996)
- 8P/Tuttle = [2B0BEAE8-26E1-4CEC-A82D-76650748F9C8](https://celestia.mobi/resources/item?item=2B0BEAE8-26E1-4CEC-A82D-76650748F9C8)
- 9P/Tempel = [ECF21CB6-222D-4A36-9994-D7BAF44F41D7](https://celestia.mobi/resources/item?item=ECF21CB6-222D-4A36-9994-D7BAF44F41D7)
- 12P/Pons–Brooks = [BF868BB1-4FCC-CFE9-2E52-5CCAACF86783](https://celestia.mobi/resources/item?item=BF868BB1-4FCC-CFE9-2E52-5CCAACF86783)
- 19P/Borrelly = [D62CABC9-3F7E-4ED2-A5C3-DB300CAC3408](https://celestia.mobi/resources/item?item=D62CABC9-3F7E-4ED2-A5C3-DB300CAC3408)
- 22P/Kopff = [1EDF2AAB-BAAE-C7D8-5AE2-D67E935AACA1](https://celestia.mobi/resources/item?item=1EDF2AAB-BAAE-C7D8-5AE2-D67E935AACA1)
- 26P/Grigg–Skjellerup = [2FD52B47-4FF4-4417-8B3F-6A79ED34C40B](https://celestia.mobi/resources/item?item=2FD52B47-4FF4-4417-8B3F-6A79ED34C40B)
- 39P/Oterma = [DB0BBBA8-E353-49FF-865A-02010E544E46](https://celestia.mobi/resources/item?item=DB0BBBA8-E353-49FF-865A-02010E544E46)
- 67P/Churyumov–Gerasimenko = [77C71308-85D7-5095-47B8-AF50E716CB3D](https://celestia.mobi/resources/item?item=77C71308-85D7-5095-47B8-AF50E716CB3D)
- 73P/Schwassmann–Wachmann = [0CC423E1-3EAA-442C-C210-4F11CA650FBA](https://celestia.mobi/resources/item?item=0CC423E1-3EAA-442C-C210-4F11CA650FBA)
- 81P/Wild = [30B8A608-C3E4-3E95-0A96-5B73BE3FAA19](https://celestia.mobi/resources/item?item=30B8A608-C3E4-3E95-0A96-5B73BE3FAA19)
- 96P/Machholz = [645EF06A-B4F9-4D56-9F8C-9CEBDAF52806](https://celestia.mobi/resources/item?item=645EF06A-B4F9-4D56-9F8C-9CEBDAF52806)
- 97P/Metcalf–Brewington = [0818BFFB-1C0A-AAA5-3A90-AD87E802661B](https://celestia.mobi/resources/item?item=0818BFFB-1C0A-AAA5-3A90-AD87E802661B)
- 100P/Hartley = [375A913D-2FD2-7CE9-13BD-2D74A84C8CBF](https://celestia.mobi/resources/item?item=375A913D-2FD2-7CE9-13BD-2D74A84C8CBF)
- 103P/Hartley = [F1345D0D-32F2-A5DB-24F1-EE28E33832F0](https://celestia.mobi/resources/item?item=F1345D0D-32F2-A5DB-24F1-EE28E33832F0)
- 110P/Hartley = [3CFFE7AB-0947-464D-9555-E506C442A77E](https://celestia.mobi/resources/item?item=3CFFE7AB-0947-464D-9555-E506C442A77E)
- 396P/Leonard = [1C96B314-2B26-D39A-7122-7AB98EFDB3B0](https://celestia.mobi/resources/item?item=1C96B314-2B26-D39A-7122-7AB98EFDB3B0)
- 449P/Leonard = [210486A3-9DEC-91E3-3C76-B3AF005D4DD7](https://celestia.mobi/resources/item?item=210486A3-9DEC-91E3-3C76-B3AF005D4DD7)
- 450P/LONEOS = [109C57E8-204A-3815-2A69-C77D0539F033](https://celestia.mobi/resources/item?item=109C57E8-204A-3815-2A69-C77D0539F033)
- 489P/Denning = [3277BF56-D29D-482D-9CCC-BF4970ABDB01](https://celestia.mobi/resources/item?item=3277BF56-D29D-482D-9CCC-BF4970ABDB01)
- D/1993 F2 (Shoemaker–Levy 9) = [DEC2DACF-A362-BE8A-B7B4-E8ABE22C682B](https://celestia.mobi/resources/item?item=DEC2DACF-A362-BE8A-B7B4-E8ABE22C682B)
- P/2019 LD<sub>2</sub> (ATLAS) = [36987389-B91D-4A38-A78B-9DB9009EB0A5](https://celestia.mobi/resources/item?item=36987389-B91D-4A38-A78B-9DB9009EB0A5)
- P/2023 JN<sub>16</sub> (Lemmon) = [327D444C-E706-418C-9F84-2F55F7839538](https://celestia.mobi/resources/item?item=327D444C-E706-418C-9F84-2F55F7839538)

```
3DCF1DB4-7CDC-436C-80E6-056D30D0DEF1
4A311B53-D159-48C4-A950-E8E63A2B6996
2B0BEAE8-26E1-4CEC-A82D-76650748F9C8
ECF21CB6-222D-4A36-9994-D7BAF44F41D7
BF868BB1-4FCC-CFE9-2E52-5CCAACF86783
D62CABC9-3F7E-4ED2-A5C3-DB300CAC3408
1EDF2AAB-BAAE-C7D8-5AE2-D67E935AACA1
2FD52B47-4FF4-4417-8B3F-6A79ED34C40B
DB0BBBA8-E353-49FF-865A-02010E544E46
77C71308-85D7-5095-47B8-AF50E716CB3D
0CC423E1-3EAA-442C-C210-4F11CA650FBA
30B8A608-C3E4-3E95-0A96-5B73BE3FAA19
645EF06A-B4F9-4D56-9F8C-9CEBDAF52806
0818BFFB-1C0A-AAA5-3A90-AD87E802661B
375A913D-2FD2-7CE9-13BD-2D74A84C8CBF
F1345D0D-32F2-A5DB-24F1-EE28E33832F0
3CFFE7AB-0947-464D-9555-E506C442A77E
210486A3-9DEC-91E3-3C76-B3AF005D4DD7
109C57E8-204A-3815-2A69-C77D0539F033
3277BF56-D29D-482D-9CCC-BF4970ABDB01
DEC2DACF-A362-BE8A-B7B4-E8ABE22C682B
327D444C-E706-418C-9F84-2F55F7839538
1C96B314-2B26-D39A-7122-7AB98EFDB3B0
36987389-B91D-4A38-A78B-9DB9009EB0A5
```

**Comment by DaveBowman2001** [comment]
@levinli303 

**Short Name**: Non-periodic Comets
**Full Name**: Comets: Non-periodic
**Description**: These are all the comets that have orbital periods greater than 200 years in length. This include long-period, near-parabolic, and hyperbolic (Oort cloud) comets
**Parent**: Comets
**Image**:
![Image](https://github.com/user-attachments/assets/365c4927-53a0-4574-8150-74fabb24d6ae)
**Add-ons**: 
- C/1680 V1 (Newton) = [7D81BA1F-C6E7-F0E7-5434-5FD58B8A0DA3](https://celestia.mobi/resources/item?item=7D81BA1F-C6E7-F0E7-5434-5FD58B8A0DA3)
- C/1980 E1 (Bowell) = [E96E5A59-AEEF-8F44-66DD-DF45D12E3848](https://celestia.mobi/resources/item?item=E96E5A59-AEEF-8F44-66DD-DF45D12E3848)
- C/1995 O1 (Hale–Bopp) = [483993BB-5304-46E3-B5CA-4D5D03883B55](https://celestia.mobi/resources/item?item=483993BB-5304-46E3-B5CA-4D5D03883B55)
- C/1996 B2 (Hyakutake) = [0B94AC4A-F4B7-470C-B345-9E839D25675A](https://celestia.mobi/resources/item?item=0B94AC4A-F4B7-470C-B345-9E839D25675A)
- C/2012 S1 (ISON) = [34649457-0B62-A509-19ED-7CAEC369EDB8](https://celestia.mobi/resources/item?item=34649457-0B62-A509-19ED-7CAEC369EDB8)
- C/2013 A1 (Siding Spring) = [66055788-51F5-79CC-37C8-CB374DE66845](https://celestia.mobi/resources/item?item=66055788-51F5-79CC-37C8-CB374DE66845)
- C/2014 UN<sub>271</sub> (Bernardinelli–Bernstein) = [C6732E2E-11F7-235D-355D-AFED31939A1A](https://celestia.mobi/resources/item?item=C6732E2E-11F7-235D-355D-AFED31939A1A)
- C/2017 K2 (PanSTARRS) = [75EEF762-D501-FD3A-6606-C4167EF4AB40](https://celestia.mobi/resources/item?item=75EEF762-D501-FD3A-6606-C4167EF4AB40)
- C/2020 F3 (NEOWISE) = [5EA9E221-C937-D96C-1016-C501F89E91E2](https://celestia.mobi/resources/item?item=5EA9E221-C937-D96C-1016-C501F89E91E2)
- C/2021 A1 (Leonard) = [A5006779-694A-1139-D643-08BDCD0B875E](https://celestia.mobi/resources/item?item=A5006779-694A-1139-D643-08BDCD0B875E)
- C/2022 E3 (ZTF) = [8979E110-906C-46A6-BE29-29BAEF615771](https://celestia.mobi/resources/item?item=8979E110-906C-46A6-BE29-29BAEF615771)
- C/2023 A3 (Tsuchinshan–ATLAS) = [1726BFEB-C111-4A01-9952-6FA0EABBBB9C](https://celestia.mobi/resources/item?item=1726BFEB-C111-4A01-9952-6FA0EABBBB9C)
- C/2023 P1 (Nishimura) = [8909041F-AF6B-4796-B7FA-4BE3BA5D41B8](https://celestia.mobi/resources/item?item=8909041F-AF6B-4796-B7FA-4BE3BA5D41B8)
- C/2024 G3 (ATLAS) = [FCD87516-0C05-4E9A-BEC7-782200D3C4D3](https://celestia.mobi/resources/item?item=FCD87516-0C05-4E9A-BEC7-782200D3C4D3)
- C/2024 L5 (ATLAS) = [D1BAD854-C308-45B6-82EE-128BA1350756](https://celestia.mobi/resources/item?item=D1BAD854-C308-45B6-82EE-128BA1350756)
- C/2024 S1 (ATLAS) = [1EC4AFAF-022A-4461-A271-E9D083DE7D38](https://celestia.mobi/resources/item?item=1EC4AFAF-022A-4461-A271-E9D083DE7D38)

```
7D81BA1F-C6E7-F0E7-5434-5FD58B8A0DA3
E96E5A59-AEEF-8F44-66DD-DF45D12E3848
483993BB-5304-46E3-B5CA-4D5D03883B55
0B94AC4A-F4B7-470C-B345-9E839D25675A
34649457-0B62-A509-19ED-7CAEC369EDB8
66055788-51F5-79CC-37C8-CB374DE66845
C6732E2E-11F7-235D-355D-AFED31939A1A
75EEF762-D501-FD3A-6606-C4167EF4AB40
5EA9E221-C937-D96C-1016-C501F89E91E2
A5006779-694A-1139-D643-08BDCD0B875E
8979E110-906C-46A6-BE29-29BAEF615771
1726BFEB-C111-4A01-9952-6FA0EABBBB9C
8909041F-AF6B-4796-B7FA-4BE3BA5D41B8
FCD87516-0C05-4E9A-BEC7-782200D3C4D3
D1BAD854-C308-45B6-82EE-128BA1350756
1EC4AFAF-022A-4461-A271-E9D083DE7D38
```

**Comment by SpaceExplorer2008** [comment]
Category modifications.

**Description:** A catalog containing approximately 5000 minor planets (MBAs, NEOs, TNOs and centaurs). ~1600 objects have 3D models derived from light curve inversion.
**Parent:** Asteroids and NEOs

**ID:** 7ECCF5CD-1EAB-4EC0-A4A2-9CCCEBBBDEC1

@levinli303

**Comment by levinli303** [comment]
> A catalog containing approximately 4600 minor planets (MBAs, NEOs, TNOs and centaurs). ~1600 objects have 3D models derived from light curve inversion.

please only include changed fields. and always the ID of the category.

**Comment by DaveBowman2001** [comment]
@levinli303 
For future add-ons listing comets that don't have individual custom add-ons, similar to the [Asteroids: Minor Planet Catalog](https://celestia.mobi/resources/category?category=7ECCF5CD-1EAB-4EC0-A4A2-9CCCEBBBDEC1) category

**Short/Full Name**: Comet Catalog
**Description**: A large catalog containing approximately 4,000 known comets
**Parent**: Comets
**Image**:
<img width="900" height="300" alt="Image" src="https://github.com/user-attachments/assets/ed66dada-b27c-4486-9cdd-c07f7b2ff853" />
**Add-ons**: TBA

**Comment by crowbar-progidy** [comment]
Short Name: Extragalactic Planets
Full Name: Exoplanets: Extragalactic Planets Description: A catalog of individual extragalactic planet candidates outside the Milky Way galaxy, from 1996 to now.
Parent: Exoplanets
Image: <img width="800" height="297" alt="Image" src="" />
Add-ons:
• M51-ULS-1 = C7E1C84D-4239-420A-A2D7-F380A5F94C9B
• PA-99-N2 = 4E58E264-4F54-42C6-BDA5-AD03D5FE3DC1

**Comment by levinli303** [comment]
@crowbar-progidy the image is not loading.
